WorkflowLayer/Parameters/RnaSeqAlignParameters.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WorkflowLayer
+{
+    public class RnaSeqAlignParameters
+    {
+        public RnaSeqAlignParameters()
```

Comment:
Should there be a constructor with all of the parameters?

Reply:
For each Flow, we will have different linux software and we will have a lot parameters.  I think we need to have several Parameter classes. We can have a CommonParameter as MetaMorpheus.